filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/batch/GcsBatchIoCommand.scala

```diff
@@ -99,10 +102,10 @@ case class GcsBatchDeleteCommand(
                                   setUserProject: Boolean = false
                                 ) extends IoDeleteCommand(file, swallowIOExceptions) with SingleFileGcsBatchIoCommand[Unit, Void] {
   private val blob = file.blob
-  def operation = {
+  def operation: Storage#Objects#Delete = {
```

Comment:
ToL: For consistency with the other overrides how about:

```scala
override def operation: StorageRequest[Void]
```

Or, those others could be made as specific as this one. Either way this should also have the `override` modifier too.